settings.py

```diff
@@ -21,3 +21,10 @@
 # bot avatar
 icon_default = ('https://upload.wikimedia.org/wikipedia/' +
                 'commons/thumb/7/7e/Hammer_and_sickle.svg/1024px-Hammer_and_sickle.svg.png')
+
+# Wolfram Alpha API token (optional)
+wa_token = os.getenv('wa_token', '')
+
+# Fancy message processing
+enable_message_processing = False
```

Comment:
These should also come from the env if available.

Reply:
excellent point. I've updated the PR